Lib/test/test_generators.py

```diff
@@ -2013,7 +2013,7 @@ def printsolution(self, x):
 >>> def f(): (yield bar) = y
 Traceback (most recent call last):
   ...
-SyntaxError: cannot assign to yield expression
+SyntaxError: cannot assign to yield expression. Maybe you meant '==' instead of '='?
```

Comment:
Here and elsewhere it feels jarring that the initial sentence ("cannot ...") doesn't start with a capital letter, but is followed by a period and another sentence that does.

Reply:
Ok, I can update the error message with a capital letter.

Reply:
Actually, on second thought, the capital letter technically is the "S" in `SyntaxError` no?